### DISCUSSION TITLE
support for activity inside containers?

### DISCUSSION THREAD

**Discussion by ghost** [sysstat/sysstat#318] [Q&A] [answered]
Sorry if this is a dumb question!  
sar/sadf works so well at the host OS level to detect resource exhaustion, it would be nice to be able to do similar inside docker containers. 
Perhaps that is what #260 is about?
Alternative suggestions welcome.
Thanks.

**Comment by sysstat** [comment, marked as answer]
I don't have any opinion about this. I have never used containers (I only have a laptop at home to use and test sysstat...) Patches welcome anyway.
